62.CAD/Polyline.h

```diff
@@ -240,7 +255,18 @@ class CPolyline
 		{
 			m_sections[m_sections.size() - 1u].count += static_cast<uint32_t>(quadBeziers.size());
 		}
-		m_quadBeziers.insert(m_quadBeziers.end(), quadBeziers.begin(), quadBeziers.end());
+
+		constexpr QuadraticBezierInfo EMPTY_QUADRATIC_BEZIER_INFO = {};
+		const uint32_t newQuadBezierSize = m_quadBeziers.size() + quadBeziers.size();
+		m_quadBeziers.reserve(newQuadBezierSize);
```

Comment:
I think I already commented on this.
you can `resize` instead of `reserve` and then do `m_quadBeziers[i+oldSize].p[0]=quadBeziers[i].P0` 
instead of emplacing and getting the "lastBezier"